File: perl/read_json_file.pl
use strict;
use warnings;
use JSON;

my $file_path = './people.json';
open my $fh, '<', $file_path;

my $people = decode_json(do { local $/; <$fh> });

print "Hello, $_->{'age'} year old $_->{'first_name'}\n" for @$people;

File: lua/read_json_file.lua
local json = require("dkjson")

local filePath = "people.json"

local file = io.open(filePath, "r")
local jsonString = file:read("*a")
file:close()

local people, pos = json.decode(jsonString, 1, nil)

for _, person in ipairs(people) do
    print(string.format("Hello, %d year old %s", person.age, person.first_name))
end
